data/json/items/vehicle/engineering.json

```diff
@@ -1,4 +1,18 @@
 [
+  {
+    "id": "chain_hoist",
+    "type": "GENERIC",
+    "category": "veh_parts",
+    "name": { "str": "chain hoist" },
+    "description": "Manual lifting equipment featuring a chain and hook.  It could be used to lift heavy loads up to 1.5 metric ton.",
```

Comment:
```suggestion
    "description": "Manual lifting equipment featuring a chain and hook.  It could be used to lift heavy loads up to 1.5 metric tons.",
```